src/schema/mutation/deleteDashboard.mutation.ts

```diff
@@ -23,22 +23,29 @@ export default {
           context.i18next.t('common.errors.userNotLogged')
         );
       }
-
       const ability: AppAbility = context.user.ability;
-      if (ability.can('delete', 'Dashboard')) {
-        return await Dashboard.findByIdAndDelete(args.id);
-      } else {
-        const page = await Page.accessibleBy(ability, 'delete').where({
-          content: args.id,
-        });
-        const step = await Step.accessibleBy(ability, 'delete').where({
-          content: args.id,
-        });
-        if (page || step) {
+      if (args.hardDelete) {
```

Comment:
@ketanreliefapps not a valid arg
keep the delete, and depending on current status of the dashboard, hard delete it or not